radixdlt-core/radixdlt/src/main/resources/default.config

```diff
@@ -154,6 +154,21 @@ ntp.pool=pool.ntp.org
 # Default: 5000
 # network.p2p.ping_timeout=5000
 
+# Specifies whether this node should act as a proxy
+# Default: false
+# network.p2p.proxy.enabled=false
+
+# A list of authorized proxy nodes
```

Comment:
Good point, I think this might require more documenting work than just a comment. The easiest way to read node's address is to get it from the logs/api, but that requires a started app and we might run into a circular dependency problem (proxy needs validator's id and the validator needs proxy's id). A CLI tool could help here.